src/odemis/acq/stream/_projection.py

```diff
@@ -934,13 +945,11 @@ def _on_max_projection(self, value):
     def getBoundingBox(self):
         '''
         Get the bounding box of the whole image, whether it`s tiled or not.
+        Since the image can now be updated, the bounding box can change
+        so we need to get it from the stream directly.
```

Comment:
As mentioned in PR #2888, I think that .rect.range should also be updated when `force_image_update()` is called. Then, this code can probably stay as-is.